### PR TITLE
More Tests 

### PR DESCRIPTION
**Pull request by martham93**
simple tests with static data for VedaCollection and DataPoint. 
Vedabase simple tests will be coming soon

### CODE REVIEW

File: pyveda/tests/unit/test_datapoint.py [hunk 1]
```diff
@@ -25,28 +25,28 @@ def setUp(self):
...
-        assert dp.id == 'ae91f7df-ae37-4d31-9506-d9176f50403c' 
-        assert dp.mltype == 'classification' 
-        assert dp.dtype == 'uint8' # based on init logic and sample json
-        assert dp.label == {"building": 0}
-        assert dp.bounds == [
+        assertEqual(dp.id, 'ae91f7df-ae37-4d31-9506-d9176f50403c')
+        assertEqual(dp.mltype, 'classification')
+        assertEqual(dp.dtype, 'uint8') # based on init logic and sample json
+        assertEqual(dp.label, {"building": 0})
+        assertEqual(dp.bounds. [
```
Comment (drwelby) [inline pyveda/tests/unit/test_datapoint.py]: Needs a comma here

**Comment by drwelby** [comment]
There's some junk files in here - don't need the fetcher.log or the directory thumbnails.

File: pyveda/tests/unit/test_datapoint.py (hunk 1 above)
Comment (rjpolackwich) [reply pyveda/tests/unit/test_datapoint.py]: this should be `self.assert....`!!!

Comment (martham93) [reply pyveda/tests/unit/test_datapoint.py]: locally they run, and have `self.assert` I think that's not the most recent commit

Comment (drwelby) [reply pyveda/tests/unit/test_datapoint.py]: yeah, I just merged your branch and that looks fixed now

Comment (drwelby) [reply pyveda/tests/unit/test_datapoint.py]: merged locally, that is

**Comment by drwelby** [comment]
The directory thumbnails are hidden - `.DS_Store`

**Comment by drwelby** [comment]
Still one more `.DS_Store` hiding in `src`

**Pull request merged**